test/parallel/test-intl.js

```diff
@@ -60,14 +60,21 @@ if (!haveIntl) {
 
   // Check with toLocaleString
   var localeString = dtf.format(date0);
-  assert.equal(localeString, 'Jan 70');
+  assert.equal(localeString, common.engineSpecificMessage({
+    v8: 'Jan 70',
+    chakracore: '\u200EJan\u200E \u200E70'
```

Comment:
Sounds good to me then.

Sent from Outlook Mobile<https://aka.ms/blhgte>



On Sat, Apr 9, 2016 at 7:38 AM -0700, "Jianchun Xu" <notifications@github.com<mailto:notifications@github.com>> wrote:


In test/parallel/test-intl.js<https://github.com/nodejs/node-chakracore/pull/54#discussion_r59116119>:

> @@ -60,14 +60,21 @@ if (!haveIntl) {
>
>    // Check with toLocaleString
>    var localeString = dtf.format(date0);
> -  assert.equal(localeString, 'Jan 70');
> +  assert.equal(localeString, common.engineSpecificMessage({
> +    v8: 'Jan 70',
> +    chakracore: '\u200EJan\u200E \u200E70'


This issue is known and being discussed at ChakraCore and TC39:

Microsoft/ChakraCore#599<https://github.com/Microsoft/ChakraCore/issues/599>
tc39/ecma402#28<https://github.com/tc39/ecma402/issues/28>

Patching it enables rest of the test to run and gives us more test coverage.

-
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/nodejs/node-chakracore/pull/54/files/2fc922aaa3e31e99fca87e1b46817d41c48986f5..c042de3faa269497ca0b8406706476dd717ff0e4#r59116119>